File: core/src/main/java/com/lomekwi/cave/util/i18n/I18N.java
package com.lomekwi.cave.util.i18n;

@Deprecated
public class I18N {
    //TODO
    @Deprecated
    public static String i18n(String key) {
        return key;
    }
    @Deprecated
    public static String[] i18n(String... keys) {
        for(int i=0; i<keys.length; i++){
            keys[i] = i18n(keys[i]);
        }
        return keys;
    }
}

File: core/src/main/scala/com/lomekwi/cave/util/i18n/I18N.scala
package com.lomekwi.cave.util.i18n

object I18N {
  //TODO
  def i18n(key: String): String = {
    key
  }
  def i18n(keys: String*): Array[String] = {
    keys.map(i18n).toArray
  }
}
